eclipse-theia/theia
# documentation: update publishing documentation

#### What it does


The following commit updates the `publishing` documentation to include steps on how to perform a GitHub release for the project including adding a reference to the changelog for the specific release version.

#### How to test


- Verify the contents of the [rendered version](https://github.com/eclipse-theia/theia/blob/2507eecfbe21dde7038bcea217f9be50f8ba1897/doc/Publishing.md#create-a-release).

#### Review checklist


#### Reminder for reviewers

- as a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)

Signed-off-by: vince-fugnitto <vincent.fugnitto@ericsson.com>
